assets/scripts/users/authentication.js

```diff
@@ -49,7 +49,7 @@ export function isSignedIn () {
 
 export function goReloadClearSignIn () {
   store.dispatch(clearSignInData())
-  saveSignInDataLocally()
+  window.localStorage.clear()
```

Comment:
Why do we want to clear all localstorage here? The function it replaces only removes one localstorage item.

Reply:
Change this to remove `LOCAL_STORAGE_SIGN_IN_ID`